src/physics/base/PhysicsStepUtils.i.hh

```diff
@@ -80,36 +81,111 @@ calc_tabulated_physics_step(const MaterialTrackView& material,
 
 //---------------------------------------------------------------------------//
 /*!
- * Calculate energy loss over the given "true" step length.
+ * Calculate mean energy loss over the given "true" step length.
+ *
+ * See section 7.2.4 Run Time Energy Loss Computation of the Geant4 physics
+ * manual. See also the longer discussions in section 8 of PHYS010 of the
+ * Geant3 manual (1993).
+ *
+ * Energy loss rate (stopping power) is a function of differential cross
+ * section: the integral of low-energy secondaries (below \c T) produced as a
+ * function of energy:
```

Comment:
The wording here is a little confusing to me.

Reply:
Yep. I've rewritten in an attempt to improve clarity.